tests/test_devices.py

```diff
@@ -60,6 +59,135 @@ def test_device_constructor_with_config(client):
+        # Test
+        assert dict(alarm_device.items(q.INPUTS, status=False)) == {}
+
+
+class TestItemSectors:
+    def test_without_status(self, alarm_device):
+        """Verify that querying items without specifying a status works correctly"""
+        alarm_device.connect("username", "password")
+        sectors = {
+            0: {"element": 1, "excluded": False, "id": 1, "index": 0, "name": "S1 Living Room", "status": True},
+            1: {"element": 2, "excluded": False, "id": 2, "index": 1, "name": "S2 Bedroom", "status": True},
+            2: {"element": 3, "excluded": False, "id": 3, "index": 2, "name": "S3 Outdoor", "status": False},
+        }
+        # Test
+        alarm_device.update()
+        assert dict(alarm_device.items(q.SECTORS)) == sectors
+
+    def test_with_status(self, alarm_device):
+        """Verify that querying items with specifying a status works correctly"""
+        alarm_device.connect("username", "password")
+        # Test
+        alarm_device.update()
+        assert dict(alarm_device.items(q.SECTORS, status=False)) == {
+            2: {"element": 3, "excluded": False, "id": 3, "index": 2, "name": "S3 Outdoor", "status": False}
+        }
+
+    def test_without_inventory(self, alarm_device):
+        """Verify that querying items without inventory populated works correctly"""
+        alarm_device._inventory = {}
+        # Test
+        assert dict(alarm_device.items(q.SECTORS, status=False)) == {}
+
+    def test_with_empty_query(self, alarm_device):
+        """Verify that querying items with empty query works correctly"""
+        alarm_device._inventory = {q.SECTORS: {}}
+        # Test
+        assert dict(alarm_device.items(q.SECTORS, status=False)) == {}
+
+
+class TestItemAlerts:
+    def test_without_status(self, alarm_device):
+        """Verify that querying items without specifying a status works correctly"""
+        alarm_device.connect("username", "password")
+        alerts = {
+            0: {"name": "alarm_led", "status": 0},
+            1: {"name": "anomalies_led", "status": 1},
+            2: {"name": "device_failure", "status": 0},
+            3: {"name": "device_low_battery", "status": 0},
+            4: {"name": "device_no_power", "status": 0},
+            5: {"name": "device_no_supervision", "status": 0},
+            6: {"name": "device_system_block", "status": 0},
+            7: {"name": "device_tamper", "status": 1},
+            8: {"name": "gsm_anomaly", "status": 0},
+            9: {"name": "gsm_low_balance", "status": 0},
+            10: {"name": "has_anomaly", "status": False},
+            11: {"name": "input_alarm", "status": 0},
+            12: {"name": "input_bypass", "status": 0},
+            13: {"name": "input_failure", "status": 0},
+            14: {"name": "input_low_battery", "status": 0},
+            15: {"name": "input_no_supervision", "status": 0},
+            16: {"name": "inputs_led", "status": 2},
```

Comment:
Side note (let's change it another PR): should we handle `inputs_led` exactly as `anomalies_led`? If we should, (so it's a `sensor` and not a `binary_sensor`, let's take a note for another PR.